bundles/org.openhab.io.openhabcloud/src/main/java/org/openhab/io/openhabcloud/internal/CloudClient.java

```diff
@@ -135,6 +137,12 @@ public class CloudClient {
      */
     private final Backoff reconnectBackoff = new Backoff();
 
+    /*
+     * Delay reconnect scheduler pool
+     * 
+     */
+    protected final ScheduledExecutorService scheduler = ThreadPoolManager.getScheduledPool("DelayConnect");
```

Comment:
Is there a need to create a new thread pool or can the common thread pool be used ?
How many threads do we expect to be started in parallel ? In case of several, I can understand the need of a dedicated thread pool.

Reply:
@kaikreuzer : WDYT ?

Reply:
Maybe @digitaldan ?

Reply:
>Maybe @digitaldan ?

 I'm on a plane with limited access to the interwebs, so I'm not able to quickly run through the code as normal.  I wanted to check if calling connect ends up running the client on that thread until disconnected or if the socket client used its own thread.  It looks like `io.socket.client.Socket` does do something with its own thread management ?  So i am assuming calling connect here should return rather quickly.   If true, i think that part looks good here.

>How many threads do we expect to be started in parallel ?

Only one, and it should not happen that frequently, and if my assumption above is correct, it would be a short lived thread.  I agree a common thread pool would make sense if available.

Reply:
So you could rather use `ThreadPoolManager.THREAD_POOL_NAME_COMMON` as thread pool name.

As this binding is probably used by almost all openHAB users, we have to be sure about any change.

Reply:
@lsafelix75 : you are welcome to discuss here with us and explain your arguments.